### ISSUE TITLE
Saving sample image to file rather than displaying in terminal

### ISSUE THREAD

**Issue by publcif** [jamesrhester/ImgCIFHandler.jl#9]
Option to save the sample image would be useful when creating e.g. html versions of the check report.

An example is at https://github.com/publcif/ImgCIFHandler.jl

NB I've not made a pull request because that fork contains other changes - i.e. work in progress... (very slow progress as I'm a novice w.r.t. Julia)

**Comment by jamesrhester** [comment]
Yep, that looks like an excellent improvement, would it make sense for the saved file to be stored in addition to the terminal output rather than instead of? Anyway, I'd suggest getting into the habit of creating a separate branch for each distinct bit of work, then the pull request system works really well. You could fork a new branch right now in your own repository for the png image save, then just delete the other work from that branch and create the pull request. The commit history might look a bit messy but the final diff will be good.

Also, if this tool ends up getting used in the background at the IUCr for checking imgCIF files, feel free to suggest/code an alternative output format (JSON? HTML?) for ease of communication to your front end.

**Comment by publcif** [comment]
Thanks for the advice re. branches and pull requests. I started again with a new fork and created a branch for a pull request (I'm also very much a novice w.r.t. github collaboration!)

For now I've kept the png output as an alternative to displaying the image in the terminal, rather than providing both options, as the IUCr use case will probably be to simply grab the stdout text and include it in an html pre wrapper along with an img element to display the png (at least in the initial stages of this project).
